crates/store/src/filesystem/utility.rs

```diff
@@ -0,0 +1,42 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: AGPL-3.0-only
+
```

Comment:
I like to avoid grab-bag miscellaneous modules, I'd say we should either find a descriptive place for these items or, failing that, just put them in mod.rs.